### DISCUSSION TITLE
the trouble after updating Monterey & Ventura reboot

### DISCUSSION THREAD

**Discussion by heikebotaru** [CobanRamo/Lenovo-IdeaPad-S340-15IIL#9] [Q&A]
Hello CobanRamo,
 Thank you for your help the other day. 
When I reboot, the keyboard is not recognised. So I can't type my password. If I press and hold the power button and then boot, the keyboard is recognised. This problem has occurred since I upgraded to Monterey. How is it with your Ideapad?

**Comment by CobanRamo** [comment]
The keyboard & TouchPad no longer respond in the OC menu after the reboot from the Monterey, correct?
This only happens with Monterey, all other operating systems do not have this.
I can only tell you that it is due to Monterey & OpenCanopy.efi.
I have already forwarded this to the developer but apparently they do not know at the moment what the problem is.
I am also waiting.

**Reply by heikebotaru** [reply]
Hi, CobanRamo.
Thank you for your reply.
It's bother for me to hold down the power button every time booting up. I hope this will be fixed in the next version.

**Comment by CobanRamo** [comment]
there is finally a solution for our dualboot friends, just tested it extensively.
thanks to our friend https://github.com/diepeterpan the problem was fixed.

A pull request was started to make it public in the next release.

Tests with the kext & bootarg from here.
https://github.com/acidanthera/bugtracker/issues/1838#issuecomment-1256948487
